components/layout/SideNav.tsx

```diff
@@ -171,12 +201,15 @@ const groupToggleDocLinkStyles = css`
   margin: 6px 0;
   font-size: 14px;
   line-height: 140%;
-  font-family: Roboto;
+  font-family: ${ROBOTO};
   color: #52627b;
-  padding: 4px 0 4px 10px;
+  padding: 4px 20px 4px 10px;
+  border-left: 1px solid transparent;
+  margin-left: -1px;
 
   &:hover {
-    color: ${ORANGE_PRIMARY};
+    border-left: 1px solid ${GREY_6};
```

Comment:
This wasn't in Kenny's mock-ups, but since the normal border color is the same as the hover background color, it felt like it needed something. Open to discussion!